services/blockchain-connector/methods/tests/decoder.js

```diff
@@ -16,8 +16,9 @@
 const {
 	decodeBlock,
 	decodeTransaction,
-	decodeEventPayload,
+	decodeSubscriptionEventPayload,
 	decodeResponse,
+	decodeEventPayload,
```

Comment:
Please name the methods appropriately based on what they do.
```suggestion
	decodeEvent,
```

Reply:
applied.